sippy-ng/src/tests/TestTable.js

```diff
@@ -322,6 +342,37 @@ function TestTable(props) {
         </div>
       ),
     },
+    delta_from_average: {
+      field: 'delta_from_average',
+      headerName: 'Delta_From_Average',
```

Comment:
These should be human readable, so headerName maybe just Delta or "Delta From Avg"? We've got a screen real estate issue I think so brief is going to be important, and then tooltips for a full description perhaps.